packages/reactive-dict/reactive-dict.js

```diff
@@ -124,6 +126,31 @@ _.extend(ReactiveDict.prototype, {
     if (_.has(self.keys, key)) oldValue = parse(self.keys[key]);
     return EJSON.equals(oldValue, value);
   },
+  
+  all: function() {
+    this.allDeps.depend();
+    var ret = {};
+    _.each(this.keys, function(value, key) {
+      ret[key] = parse(value);
+    });
+    return ret;
+  },
+  
+  clear: function() {
+    var self = this;
+    
+    var oldKeys = self.keys;
+    self.keys = {};
+    
+    self.allDeps.changed();
+    
+    _.each(oldKeys, function(value, key) {
```

Comment:
I have not actually reviewed the diff (and I personally still generally believe that Meteor would be better with less APIs and shorter docs [one of the things, say, people like about React]), but the answer to your specific question is -- yes, autoruns will only re-run once on the next flush cycle.